packages/graphql/tests/integration/directives/auth/roles.int.test.ts

```diff
@@ -111,45 +119,14 @@ describe("auth/roles", () => {
                 }
             `;
 
-            const neoSchema = new Neo4jGraphQL({ typeDefs, config: { jwt: { secret } } });
-
-            try {
-                const req = createJwtRequest(secret);
-
-                const gqlResult = await graphql({
-                    schema: await neoSchema.getSchema(),
-                    source: query,
-                    contextValue: { driver, req, driverConfig: { bookmarks: session.lastBookmark() } },
-                });
-
-                expect((gqlResult.errors as any[])[0].message).toBe("Forbidden");
-            } finally {
-                await session.close();
-            }
-        });
-
-        // This tests reproduces the security issue related to authorization without match #195
-        test.skip("should throw if missing role on type definition and no nodes are matched", async () => {
```

Comment:
Why has this test been deleted? Although skipped, it was left in on purpose to give a clear test case for #195 and highlight that it does need to be addressed at some point. We can just decorate it to get ESLint to ignore it if need be. 🙂

Reply:
Sure its added back!